### DISCUSSION TITLE
What happened to /gridpoints?

### DISCUSSION THREAD

**Discussion by HilbertIdeals5** [weather-gov/api#487] [Q&A] [answered]
I want to quickly gather weather data for a personal project, where I map out forecasts across the U.S. I noticed that forecasts are assigned to gridpoints, and I found on the API documentation that you can use the /gridpoints endpoint to get a full list of them.

But for some reason, api.weather.gov/gridpoints doesn't work. Why is that? It would be much more convenient to go through every gridpoint in one go, without having to repeat calls for every single city.

**Comment by scadergit** [comment]
The API doesn't support a batch effort at this time.

**Comment by StephenClouse** [comment, marked as answer]
I'm curious where you saw this documentation on /gridpoints, because that has definitely never existed or been supported.

In any case, there are approximately 2.5 million gridpoints, so we won't be offering a full enumeration of them anytime soon.

**Reply by HilbertIdeals5** [reply]
I found it here: https://weather-gov.github.io/api/gridpoints
In the first section. Although with what you said about there being too many gridpoints, I probably just misread it....

I also just discovered that there's a list of forecast offices in the documentation itself, so my bad 😓

**Reply by HilbertIdeals5** [reply]
The only issue I have is that I don't know what coordinate points there are for each forecast office. If I do http://api.weather.gov/offices/RAH, I get links to zones and counties, but not to forecasts; and if I try something like http://api.weather.gov/gridpoints/RAH/1,1, I just get an error.

I want to take full advantage of this API and not miss any forecast data (I'm collecting from forecastGridData). So would this be too high of an expectation for the service?

**Reply by scadergit** [reply]
You could argue that should be written without the slash as it was a general reference and not a specific example or speciation. 

Are you wanting the forecast data for every grid? If you only want specific grids, you could use the /points endpoint to convert lat/lon to a specific grid.
